controllers/model/grafanaConfig.go

```diff
@@ -27,10 +27,10 @@ func GrafanaConfig(cr *v1alpha1.Grafana) (*v1.ConfigMap, error) {
 
 	configMap.Data = map[string]string{}
 	configMap.Data[constants.GrafanaConfigFileName] = config
-	return configMap, nil
+	return configMap
 }
 
-func GrafanaConfigReconciled(cr *v1alpha1.Grafana, currentState *v1.ConfigMap) (*v1.ConfigMap, error) {
+func GrafanaConfigReconciled(cr *v1alpha1.Grafana, currentState *v1.ConfigMap) *v1.ConfigMap {
```

Comment:
never returns an error so not needed right?

Reply:
Yes exactly